stacks/container-apps/play.prx.org.yml

```diff
@@ -93,7 +95,7 @@ Resources:
           Values:
             - play.*
       ListenerArn: !Ref PlatformALBHTTPSListenerArn
-      Priority: !Join ["", [!Ref PlatformALBListenerPriorityPrefix, "00"]]
+      Priority: !Join ["", [!Ref PlatformALBListenerPriorityPrefix, "30"]]
```

Comment:
Why change this?

Reply:
If the host-only rule stays at a lower priority it will continue to catch all traffic